build/parseChangelog.c

```diff
@@ -44,7 +44,8 @@ static int dateToTimet(const char * datestr, time_t * secs, int * date_words)
     struct tm time, ntime;
     const char * const * idx;
     char *p, *pe, *q, *date, *tz;
-    char tz_name[10];               /* name of timezone (if extended format is used) */
+    char *tz_name = NULL;               /* TZ database name (if extended format is used) */
```

Comment:
Similar to my other database complaints, the comment change seems quite questionable. 
It's not a name of the database as the comment now indicates, but the name of the timezone. No comment change is necessary, and in fact this never really needed that comment to begin with because the variable name says it all. And that we're wasting time discussing something like this is a fine example of superfluous comments being bad.

Reply:
I used "TZ database name" because I read it in several sources (e.g https://en.wikipedia.org/wiki/List_of_tz_database_time_zones or some others). So it looks correct for me.

Reply:
Yes I can see it's used as a column name in that list, as a shorthard for "name *in* timezone database". But "foo database name" generally refers to the name of the database itself, not a value in it. If I were talking about "rpm database name", you wouldn't think of individual package names would you?

Reply:
Oh and BTW this line overflows to a second line for no good reason at all. Lines should be kept under 80 characters when at all possible. The new error message line, and also the older "first part of year..." comment break this rule, for no good reasons.

Reply:
Also, *this* string is just something in the spec. We don't know if its anything resembling a valid timezone name, it's just what we *expect* it to be. And certainly it doesn't come from any database. So this turns what was originally a superfluous but harmless comment into a misleading one. If you feel compelled to change it, then just *remove* the comment please because it serves absolutely no purpose. 

What would deserve a comment however is consulting the "magic" tzname variable.